### PR TITLE
[Fix] Update manager dashboard UI

### LINKED ISSUE #11575: 🐛 Manager dashboard review change requests
## 🐛 Bug

A later review from Josh included some requested changes to the manager dashboard.

## ✅ Acceptance Criteria

- [x] The dropdown caret on the role switcher should be x.85 in width/height
- [ ] ~The top/bottom padding on chips should be x.05~ #11585
- [x] Bullets all over the page should be base(black.lighter)
- [x] Metadata text (the stuff under each request title) should be base(black.light)
- [ ] ~Padding between the request title/actions block and its metadata should be x.25~ #11587
- [x] The overall spacing inside the request accordion is a bit inconsistent - everything should be x1 apart
- [x] The magnifying glass should be indented on both mobile (x1) and desktop (x1.5)
- [x] The final item in the request accordion shouldn't have bottom padding, as the card already provides it
- [x] This change is totally on me, but all the gray dividers on the page should be updated to base:all(gray.light); I found the lighter color wasn't contrasty enough
- [x] The dropdown in the role switcher should indicate the active item (see attached image)
- [x] On mobile, the actions below each request title should stack by default, and the bullets should be hidden

### PR DESCRIPTION
**Pull request by petertgiles**
🤖 Resolves #11575

## 👋 Introduction

This branch corrects (almost) all the items from Josh's design review of the manager dashboard page.

## 🕵️ Details

Note that one AC was not done:
>  The final item in the request accordion shouldn't have bottom padding, as the card already provides it

I think it's already correct:
image

## 🧪 Testing

1. Uncomment the loader in router.tsx to enable the manager dashboard page
2. Visit /en/manager/dashboard

### CODE REVIEW

**Review by yonikid15 — CHANGES_REQUESTED** [review]
Looking good so far! :+1: 

I found one thing: The preview list padding on the bottom is still larger than it needs to be (x1). Dropping it to x.5 should keep it balanced with the accordion heading at the top.
image

**Comment by petertgiles** [comment]
Ah, that was the last one, thanks!
[The final item in the request accordion shouldn't have bottom padding…](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11667/commits/c579ba3ecbfdc92dffaa550d7f56b520d5ca3289)

**Comment by petertgiles** [comment]
Looks like x0.25 is what Josh was looking for.
image

File: apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx [hunk 1]
```diff
@@ -125,6 +125,8 @@ const ManagerDashboard = ({ userQuery }: ManagerDashboardProps) => {
                       setAccordionItems(newValue);
                     }}
                     type="multiple"
+                    // we don't need that fat padding in the accordion inside the task card
+                    data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](0)"
```
Comment (yonikid15) [inline apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx:129]: I think you want this to be x.5 so it matches the space at the top of the box.
image


```suggestion
                    data-h2-padding-bottom="base:selectors[>.Accordion__Item > .Accordion__Content](x.5)"
```

Another option could be to update the TaskCard.Item bottom padding that wraps everything, but this works as well.

**Review by yonikid15 — CHANGES_REQUESTED** [review]

**Review by yonikid15 — APPROVED** [review]
LGTM :tada:

**Pull request merged**